src/lib/popover/popover.component.ts

```diff
@@ -223,48 +259,54 @@ export class SatPopover implements OnInit, OnDestroy {
 
   constructor(
     private _focusTrapFactory: FocusTrapFactory,
+    private _anchoringService: SatPopoverAnchoringService,
```

Comment:
Note, where before the `SatPopoverAnchor` made the `_anchoringService` available publicly, here I made it private. It wasn't obvious to me why it was public in the first place, other than perhaps facilitating tests. I added a workaround to the tests, which I'll highlight below, to allow this to remain private.